### ISSUE TITLE
State of content-addressed nix/nixpkgs in 2024

### ISSUE THREAD

**Issue by katexochen** [tweag/nix-hour#81]
There was a series of [blog post in the past ](https://www.tweag.io/blog/2021-12-02-nix-cas-4/) but I haven't heard about it in long.

I'd love to see what is possible today, and what is needed to get the ecosystem into a content-addressed future.

Not sure how well this fits the nix-hours format.

**Comment by phanirithvij** [comment]
Yes! if possible get @ericson2314 and discuss it, if/whenever nix-hour s2 comes.

**Comment by Ericson2314** [comment]
I've been thinking a lot about how to wrap this experiment up and get things out in the wild lately :) Except at least a little bit at my Planet Nix talk!
